src/Nettle.jl

```diff
@@ -1,7 +1,8 @@
 module Nettle
 
-if isfile(joinpath(Pkg.dir("Nettle"),"deps","deps.jl"))
-   include("../deps/deps.jl")
+const depfile = normpath(joinpath(dirname(@__FILE__), "..", "deps", "deps.jl"))
```

Comment:
Do you know if I can squash commits once they've been pushed into a PR? I probably don't need 3 separate commits for this.